index.js

```diff
@@ -161,7 +150,7 @@ WuiDom.prototype.removeChild = function (child) {
 
 	this.rootElement.removeChild(child.rootElement);
 	this._childrenList.splice(siblingIndex, 1);
-	if (this._childrenMap.hasOwnProperty(child._name)) {
+	if (child._name in this._childrenMap) {
```

Comment:
The `in` operator traverses the prototype chain, something you may want to keep in mind.